src/platform/SaveBootReasonDFUSuit.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_SAVE_BOOT_REASON_H__
+#define NRF_SAVE_BOOT_REASON_H__
+
+inline int SoftwareBootReasonSUIT __attribute__((section(".noinit")));
```

Comment:
Please move it inside the Reboot.cpp file.

Btw. We can't have a Nordic sepecific file in the `src/platform` file anyway.

Btw2. Do we need to add a `SUIT` in the name?